server/src/controllers/users.controller.ts

```diff
@@ -87,6 +87,8 @@ export async function updateUser(req: express.Request, res: express.Response) {
 	updatedUser.data.updated = new Date();
 	const { error } = await tryPromise(updatedUser.data.save());
 
+	if (error) throw error
+
```

Comment:
Might want to remove this line...

tbf, there should be very little difference in what happens due to our top level error handler. But, this is how you could deal with errors if you aren't handling them with an if statement. They get passed up to the handler in index.ts